src/main/java/cz/cvut/kbss/study/rest/PatientRecordController.java

```diff
@@ -62,15 +57,48 @@ public List<PatientRecordDto> getRecords(
 
     @PreAuthorize(
             "hasRole('" + SecurityConstants.ROLE_ADMIN + "') or @securityUtils.isMemberOfInstitution(#institutionKey)")
-    @GetMapping(value = "/export", produces = MediaType.APPLICATION_JSON_VALUE)
-    public List<PatientRecord> exportRecords(
+    @GetMapping(value = "/export", produces = {MediaType.APPLICATION_JSON_VALUE, Constants.MEDIA_TYPE_EXCEL})
+    public ResponseEntity<?> exportRecords(
             @RequestParam(name = "institution", required = false) String institutionKey,
-            @RequestParam MultiValueMap<String, String> params,
+            @RequestParam(required = false) MultiValueMap<String, String> params,
             UriComponentsBuilder uriBuilder, HttpServletResponse response) {
+
+        String exportType = Optional.ofNullable(params)
```

Comment:
To me this is ok, although i was expecting something like:
```
    public ResponseEntity<?> exportRecords(@RequestHeader("Accept") String acceptHeader) throws Exception {

        if (acceptHeader.contains(MediaType.APPLICATION_JSON_VALUE)) {
            return exportAsJson();
        } else if (acceptHeader.contains("application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")) {
            return exportAsExcel();
        } else {
            return ResponseEntity.status(HttpStatus.NOT_ACCEPTABLE).body("Unsupported media type");
        }
    }
```

Reply:
The parameter is ok for now ... later maybe we should support both (parameter and accept-header)

Reply:
DO NOT FIX NOW

Reply:
@blcham 
I implemented extraction of export type from the http to consider accept header. `exportType` query parameter takes precedence over the accept header type. see commit [1918810 - Consider Accept header when extracting exportType from request...](https://github.com/kbss-cvut/record-manager/pull/52/commits/1918810357e24a9dfa848ca4b1c80d9d0c20ffa1)

Reply:
@kostobog perfect !